test/index.test.js

```diff
@@ -427,4 +429,40 @@ test('has @add-mixin alias', async () => {
   await run('@define-mixin a { a: 1 } @add-mixin a', 'a: 1')
 })
 
+test('runs after plugin declared earlier', async () => {
```

Comment:
Strange. Does this test fail if you do not change the `index.js`?

Reply:
No, it passes in both cases. But I wanted to make sure that such a scenario is covered by tests and protected from refactoring.